pandas/core/generic.py

```diff
@@ -1892,6 +1896,30 @@ def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
             Optional specifying the datatype for columns. The SQL type should
             be a SQLAlchemy type, or a string for sqlite3 fallback connection.
 
+        Returns
+        --------
+            None
+
+        See Also
+        --------
+        pandas.io.sql.to_sql : this function will be called.
+
+        Examples
+        --------
+        >>> import pandas as pd
+        >>> from sqlalchemy import create_engine
+        >>> engine = create_engine('sqlite:///example.db', echo=False)
+        >>> gen_names = lambda ids: ["User" + str(x) for x in ids]
+        >>> gen_users = lambda ids: {"id": ids, "name" : gen_names(ids)}
+        >>> df=pd.DataFrame(gen_users(list(range(3))))
```

Comment:
maybe it's a bit more straightforward to generate dataframe without other functions:
i.e. df=pd.DataFrame(['User 1', 'User 2', 'User 3'])